go-packages/meep-model/validator.go

```diff
@@ -33,8 +38,54 @@ const (
 	ValidatorStatusError   = "SCENARIO-ERROR"
 )
 
+const (
+	REGEX_NAME               = `^(([a-z0-9][-a-z0-9.]*)?[a-z0-9])+$`
+	REGEX_FULL_NAME          = `^(([a-z0-9][-a-z0-9.]*)?[a-z0-9])+$`
```

Comment:
why having REGEX_NAME and REGEX_FULL_NAME being the same?

Reply:
Good catch, I will consolidate in a single REGEX.